packages/pagination/README.md

```diff
@@ -99,6 +99,23 @@ return (
 );
 ```
 
+## createInfiniteScroll
+```tsx
+export default function App() {
+  // fetcher: (page: number) => Promise<T[]>
+  const [pages, asLoader, { end }] = createInfiniteScroll(fetcher);
+  return (
+    <div>
+      <For each={pages()}>
+        {item => <h4>{item}</h4>}
+      </For>
+      <Show when={!end()}>
+        <h1 use:asLoader>Loading...</h1>
```

Comment:
In the defined types the name of the directive is `loaderDirective`, and here `asLoader` is used, which ts won't like.
```ts
interface Directives {
  loaderDirective: boolean
}
```

Reply:
forgot to rename 😅